playbook/app/pb_kits/playbook/pb_form_group/docs/index.js

```diff
@@ -1,5 +1,6 @@
 export { default as FormGroupDefault } from './_form_group_default.jsx'
 export { default as FormGroupButton } from './_form_group_button.jsx'
+export { default as FormGroupFullWidth } from './_form_group_full_width.jsx'
 export { default as FormGroupDatePicker } from './_form_group_date_picker.jsx'
 // export { default as FormGroupTypeahead } from './_form_group_typeahead.jsx'
```

Comment:
Remove this?

Reply:
@megantrimble I just saw what was being referenced here (green highlight threw me off). @coleerikson is meaning line 5 here which is commented out.

Reply:
Gotcha, yea that was already in there but yes I can remove it!